src/components/Navbar.jsx

```diff
@@ -1,19 +1,22 @@
 import { Link } from 'react-router-dom';
 import './styles/Navbar.css';
+import { FaUser } from 'react-icons/fa'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
- [ ] I tried to run your project locally but it is giving me the following error. it means that this line is causing this error. kindly consider fixing it so that the reviewer can see your project and evaluate it.
error:
<img width="872" alt="Screenshot 2023-06-21 at 9 39 24 PM" src="https://github.com/iamsjunaid/Bookstore/assets/96683843/cf9c9fad-ba9f-4d37-8a0e-9bb275303943">

Reply:
Thanks for the feedback @Zuhaib042😊. I have fixed the issue.